docs/general_usage.md

```diff
@@ -33,6 +33,45 @@ The CLI will attempt to load credentials in the following order of priority:
 1. From environment variables, if set
 1. From the active profile
 
+## Subaccounts
+
+Instructions on how to create a Twilio subaccount can be found in the [Twilio docs](https://support.twilio.com/hc/en-us/articles/360011348693-View-and-Create-New-Twilio-Subaccounts).
+
+### API Commands
+
+For non-`twilio api:core` commands, please use profiles (via `twilio login` to create a new profile or `twilio profiles` to manage existing credentials).
+
+You can’t use the token created by `twilio login` to access subaccounts as it creates a Standard key which is scoped for master account only. An appropriate API key can be created from a Master Account here: https://www.twilio.com/console/project/api-keys/create.
+
+Instead of using `twilio login`, you must use environment variables if you want access to subaccounts. There are two options (Master API Key, or Master Account SID and Auth Token).
```

Comment:
Don't think this is needed here. Env vars are not needed for non-core commands. You can just `twilio login`. I think this whole section can just be reduced to the first part about using profiles to work with subaccounts for non-core commands. Just be sure to update the wording to indicate the subaccount SID should be used for the login command.